pkg/network/bridge/plugin.go

```diff
@@ -23,6 +24,8 @@ const (
 	interfaceNamePrefix = "nic_"
 	letterBytes         = "abcdefghijklmnopqrstuvwxyz0123456789"
 	assignmentTimeout   = 30 * time.Minute
+	protocolEthernet    = "Ethernet"
+	envVariableName     = "NETWORK_INTERFACE_RESOURCES_DEVICE_PLUGIN_NETWORK_BRIDGE"
```

Comment:
I think we need to use random suffix, in case user asks for two bridges, one Allocate call would overwrite another.

Reply:
sound good

Reply:
do you prefer a random suffix or a deterministic created one, e,g, the resource name, e.g.  NETWORK_INTERFACE_RESOURCES_DEVICE_PLUGIN_NETWORK_BRIDGE_RED ?

Reply:
RED saves it for multiple plugins running at the same time, but not for the potential multiple calls of Allocate (probably never happens, but better safe than sorry). Maybe both "RED" for easier debugging and random characters to be sure?

Reply:
Lets just go with a random one. o/w the name might become very long. in my code i just look for any environment var that start with: "NETWORK_INTERFACE_RESOURCES_"
so, we can probably avoid the device plugin name, and network name. and create a random suffix (8 chars would give pretty low collision probability): NETWORK_INTERFACE_RESOURCES_XXXXXXXX

Reply:
note that we modified the json a bit, so that it would include the full plugin+resource. think this is sufficient for debugging

Reply:
Done. Variable is now like this:
`NETWORK_INTERFACE_RESOURCES_FPLLNGZI='{"name":"bridge.network.kubevirt.io/br13","interfaces":[{"name":"br13-45","protocol":"Ethernet"}]}'`